torch/lib/c10d/NCCLUtils.cpp

```diff
@@ -31,4 +31,62 @@ std::string ncclGetErrorWithVersion(ncclResult_t error) {
       getNcclVersion();
 }
 
+#if NCCL_VERSION_GE(2, 7, 0)
+ncclResult_t gpuAlltoall(
+    void* sendbuff,
+    void* recvbuff,
+    size_t count,
+    size_t elem_size,
```

Comment:
nit: all other part of this file seem to use camel instead of snake naming convention.